Description: Fix preprocessing in case of empty results
 Example case:
 .
 The file "test" contains:
 .
 #@dphys-config-preprocess include include
 #@include host
 .
 and if the file "test.host" exists but is empty, this will yield in
 an error like this:
 .
 mv: cannot stat '/var/tmp/dphys-config-7962-work.1hGOSCEFfy/test.temp': No such file or directory
Author: Axel Beckert <abe@debian.org>

--- a/dphys-config
+++ b/dphys-config
@@ -642,6 +642,7 @@
     #   split junk will simply feed nonsense to case, and get caught by *)
     local PREPROC
     for PREPROC in ${PREPROCS} ; do
+      touch "${WORK}.temp"
       case "${PREPROC}" in
 
         backtick)
